core/src/smartcontracts/isi/account.rs

```diff
@@ -195,6 +195,10 @@ pub mod isi {
             let account_id = self.destination_id;
             let permission = self.object;
 
+            wsv.permission_token_ids()
+                .get(permission.id())
+                .ok_or_else(|| FindError::PermissionToken(permission.id().clone()))?;
+
```

Comment:
If there's no such token registered, it shouldn't exist in the user account anyway, since they are revoked on unregistration.
It would make sense to log such cases though, since it indicates inconsistent blockchain state.